lib/kadai-core/src/test/java/acceptance/jobs/WorkbasketCleanupJobAccTest.java

```diff
@@ -106,37 +100,6 @@ void shouldNotCleanWorkbasketMarkedForDeletionIfWorkbasketHasTasks() throws Exce
     assertThat(totalWorkbasketCount).isEqualTo(26);
   }
 
-  @WithAccessId(user = "admin")
-  @Test
-  void should_DeleteOldWorkbasketCleanupJobs_When_InitializingSchedule() throws Exception {
-
-    for (int i = 0; i < 10; i++) {
-      ScheduledJob job = new ScheduledJob();
-      job.setType(WorkbasketCleanupJob.class.getName());
-      kadaiEngine.getJobService().createJob(job);
-      job.setType(TaskRefreshJob.class.getName());
-      kadaiEngine.getJobService().createJob(job);
-      job.setType(ClassificationChangedJob.class.getName());
-      kadaiEngine.getJobService().createJob(job);
-    }
-
-    List<ScheduledJob> jobsToRun = getJobMapper(kadaiEngine).findJobsToRun(Instant.now());
-
-    assertThat(jobsToRun).hasSize(30);
```

Comment:
Why did you remove this assertion?

Reply:
I deleted the test function because I included the test case in the AbstractKadaiJobAccTest class, as required by the issue. James gave me a similar issue, which I followed: https://github.com/Taskana/taskana/pull/2470/files#diff-a9864bf32971d758536dbe9eb5f708dd09072bcc33be23bbfef2f5e7d982af22

Reply:
ok so it is fine to not test size 30 anymore?

Reply:
This test is not relevant anymore, the logic has been moved to the AbstractKadaiJobAccTest in the should_DeleteOldCleanupJobs_When_InitializingSchedule() test

Reply:
Yes that's clear. What I mean is: should the test `should_DeleteOldCleanupJobs_When_InitializingSchedule` in `AbstractKadaiJobAccTest` also `assertThat(jobsToRun).hasSize(30)` or not? And if not, why?

Reply:
I see your point now. Technically we don't need to check again if `jobsToRun` has size 30, since we know that after the for loop, 30 jobs will be initiated, 10 from TaskRefreshJob, 10 from ClassificationChangedJob, and 10 from the CleanUpJob, so asserting the `jobsToRun` is redundant

Reply:
Perfect, thanks for the explanation 👍